test/scale.time.tests.js

```diff
@@ -485,68 +485,4 @@ describe('Time scale tests', function() {
 			threshold: 0.75
 		});
 	});
-
-	it('should not throw an error if the datasetIndex is out of bounds', function() {
```

Comment:
I have read only the test title and didn't notice it was explicitly testing this method.